scripts/Manifest

```diff
@@ -8,5 +8,5 @@ REPLICATED_APP_URL=https://replicated.app
 KURL_UTIL_IMAGE=replicated/kurl-util:alpha
 KURL_BIN_UTILS_FILE=
 # STEP_VERSIONS array is generated by the server and injected at runtime based on supported k8s versions
-STEP_VERSIONS=("" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "1.15.3" "1.16.4" "1.17.13" "1.18.20" "1.19.15", "1.20.11")
-INSTALLER_YAML=
\ No newline at end of file
+STEP_VERSIONS=(0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 0.0.0 1.16.4 1.17.13 1.18.20 1.19.16 1.20.15 1.21.14 1.22.13 1.23.10 1.24.4)
```

Comment:
I don't think this should have been updated

Reply:
It's unrelated but it is helpful for the next developer working on upgrades of kubernetes.

Reply:
@emosbaugh Why do we need all the `0.0` versions?

Reply:
The relevant code is [here](https://github.com/replicatedhq/kURL/blob/7376cc9d7441f22b650d22209be9eec8ab223011/scripts/common/discover.sh#L247-L249). I needed to add the newer versions. This is placeholder for each minor version. 0.0.0 isn't necessary its just what I copied from production. See [here](view-source:https://k8s.kurl.sh/latest)

Reply:
@emosbaugh may be ok but: have you intentionally skipped 1.15.3 ?